app/scripts/views/auto-type/auto-type-select-view.js

```diff
@@ -26,6 +27,8 @@ const AutoTypePopupView = Backbone.View.extend({
         this.listenTo(Backbone, 'main-window-will-close', this.mainWindowWillClose);
         KeyHandler.onKey(Keys.DOM_VK_ESCAPE, this.escPressed, this, false, true);
         KeyHandler.onKey(Keys.DOM_VK_RETURN, this.enterPressed, this, false, true);
+        KeyHandler.onKey(Keys.DOM_VK_RETURN, this.actionEnterPressed, this, KeyHandler.SHORTCUT_ACTION, true);
+        KeyHandler.onKey(Keys.DOM_VK_RETURN, this.optEnterPressed, this, KeyHandler.SHORTCUT_OPT, true);
```

Comment:
Please add an explanation text in the view somewhere. Nobody is going to notice it otherwise.